Formula/restic.rb

```diff
@@ -16,8 +17,9 @@ class Restic < Formula
 
   def install
     ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    ENV["CGO_ENABLED"] = "1"
 
-    system "go", "run", "build.go"
+    system "go", "run", "build.go", "--enable-cgo"
```

Comment:
```suggestion
    system "go", "run", "-mod=vendor", "build.go", "--enable-cgo"
```

https://github.com/restic/restic/issues/2428
https://github.com/Homebrew/linuxbrew-core/pull/16220

Reply:
thanks!